e2e/teams.spec.ts

```diff
@@ -171,15 +189,11 @@ test.describe('Teams', () => {
       })
 
       await testAuth.step('user deletes team', async () => {
-        await goToDashboard(userPage)
-        await userPage.getByLabel('Teams').click()
-        await userPage.getByRole('link', { name: TEAM_NAME }).click()
```

Comment:
Delete the user from the current location (Team page), rather than navigating to the dashboard then back to the Teams page -- this was a hold-over from moving this step from the `afterAll` into the test